### PR TITLE
Team chest modification

### PR DESCRIPTION
**Pull request by Themostrandom**
- [ x ] This PR has been tested locally
- This change allows us to expand the team chest, which is quickly full on some maps.

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
Was this suggested at all first?

For CTF PRs you need to use spaces instead of tabs for indentation btw

File: mods/ctf/ctf_teams/team_chest.lua [hunk 1]
```diff
@@ -1,336 +1,260 @@
...
-	"default:pick_stone",
+    "default:pine_needles",
+    ".*leaves$",
+    "ctf_melee:sword_stone",
+    "default:pick_stone",
 }
 
---[[
-local item_value = {
-	["grenades:poison"] = 5,
```
Comment (farooqkz) [inline mods/ctf/ctf_teams/team_chest.lua:10]: These must not be removed.

**Review by farooqkz — CHANGES_REQUESTED** [review]

**Comment by farooqkz** [comment]
> Was this suggested at all first?
> 
> For CTF PRs you need to use spaces instead of tabs for indentation btw

Expanding teamchest space has been suggested many times. And if you remember, we had discussions about implementation of it, too. however, our discussions didn't come to an agreement. Finally you made two suggestions to see which impementation would be more popular among the players but again it was a tie regarding vote counts.

**Comment by farooqkz** [comment]
> Was this suggested at all first?
> 
> For CTF PRs you need to use spaces instead of tabs for indentation btw

You mean vice versa. right?

**Comment by LoneWolfHT** [comment]
> You mean vice versa. right?

Hah, yes.

**Pull request closed without merging**